Stratio/kind
# [CLOUD-47] Actualizar CAPG (a 1.4.0)

## Test plan

- Deploy an unmanaged cluster :heavy_check_mark:
- Check all pods are in a `Running` state :heavy_check_mark:
- Scale a `MachineDeployment`  :heavy_check_mark:
- Verify CAPG image is in the correct version:  kw get po -n capg-system -oyaml | grep image:   :heavy_check_mark:
